reframe/frontend/cli.py

```diff
@@ -494,6 +494,14 @@ def main():
         if options.purge_env:
             rt.modules_system.unload_all()
 
+        # Load the environment for the current system
+        try:
+            rt.system.preload_environ.load()
+        except EnvironError as e:
+            printer.error("failed to load current's system environment; "
+                          "please check your configuration")
+            raise
```

Comment:
Are we raising empty?

Reply:
No, this means raise the current exception. But I should remove `e`, which I don't use.